### PR TITLE
Body scanners show how much bones have been strengthened

### PR DESCRIPTION
**Pull request by DamianX**
image

:cl:
 * rscadd: Body scanners have been upgraded and can now determine increased (or decreased) bone strength.

### CODE REVIEW

File: code/game/machinery/adv_med.dm [hunk 1]
```diff
@@ -456,6 +457,11 @@
 			robot = "Prosthetic:"
 		if(e.open)
 			open = "Open:"
+		if(e.min_broken_damage != initial(e.min_broken_damage))
+			var/increase = e.min_broken_damage - initial(e.min_broken_damage)
+			var/percent_increase = round(increase / initial(e.min_broken_damage) * 100)
+			var/sign = percent_increase > 0 ? "+" : ""
```
Comment (MadmanMartian) [inline code/game/machinery/adv_med.dm]: what about a possibility for bone weakening?

Comment (DamianX) [reply code/game/machinery/adv_med.dm]: Seems to me like this should work?

Comment (MadmanMartian) [reply code/game/machinery/adv_med.dm]: Not quite. Say you've got a e.min_broken_damage of 5, and its initial 15. It'd show 67 percent, when it should show 33 if there is no minus symbol involved. Maybe change the "" to "-"?

**Comment by DamianX** [comment]
image
Changed it use raw values instead.

**Review by MadmanMartian — APPROVED** [review]

**Pull request merged**